ikaaro/folder.py

```diff
@@ -90,23 +99,17 @@ def get_document_types(self):
     def traverse_resources(self):
         yield self
         for name in self._get_names():
-            resource = self.get_resource(name)
+            try:
+                resource = self.get_resource(name)
+            except StopIteration:
+                # Log the resource abspath
+                continue
```

Comment:
Should never happen ?

Reply:
It can happen when a resource file is empty